docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -167,6 +167,8 @@ quarkus.oidc.credentials.secret=secret
 quarkus.keycloak.policy-enforcer.enable=true
 ----
 
+NOTE: By default, applications using the `quarkus-oidc` extension are marked as a `service` type application (see `quarkus.oidc.application-type`). This extension only support applications marked as such.
+
```

Comment:
May be we should say "This extensions currently supports only such `service` applications" ? To have an indirect message that we may want to support not only "service" applications in the future ?